tests/unit/providers/test_gemini_interactions.py

```
@@ -0,0 +1,784 @@
+    yield InteractionStatusUpdate(interaction_id="int_late", status="in_progress")
+    yield InteractionCompletedEvent(
+        interaction=InteractionSseEventInteraction(
+            id="int_late",
+            status="completed",
+            model="gemini-3-flash-preview",
+        ),
+    )
+
+
+@pytest.mark.asyncio
+async def test_convert_interaction_stream_emits_openai_events() -> None:
```

Ruff (0.16.2)

[warning] Too many statements (53 > 50)

(PLR0915)